src/check/decl.rs

```diff
@@ -46,6 +46,7 @@ pub fn check_lift_parameters<'a>(
         Expression::Pi(
             (pattern.clone(), Box::new(*expression)),
             Box::new(signature),
+            _level,
```

Comment:
Do we really need this?

How about introducing a `level()` member function for `Value` or `Expression` that returns the level of a type value (should not be called for non-type values, so you can panic if it's an invalid call), where for `Pi` it returns the return-type's level?

Reply:
In this way we probably won't need a member variable for `Sum`/`Pi`/`Sigma`. However, storing a value there can be a kind of cache optimization... What do you think?